Trying to merge with my matrix additions

TransientResponse opened the pull request:
I HAVE NO IDEA WHAT I'M DOING

Xlythe committed:
save cycles - append ) when saving

Xlythe committed:
bugfix

Xlythe committed:
version bump

Xlythe committed:
remove old matrix code

Xlythe committed:
add new matrix page

Xlythe committed:
separate views into own package

Xlythe committed:
bugfix

Xlythe committed:
fix preferences

Xlythe committed:
move tablet xml from generic to port

Xlythe committed:
remove deprecated code

Xlythe committed:
hide display on matrix page

Xlythe committed:
switch to matrix color buttons

Xlythe committed:
wireframe of logic

Xlythe committed:
new design should allow views in edittext

Xlythe committed:
move copying

Xlythe committed:
automatically display hints

Xlythe committed:
bugfix for tablets

Xlythe committed:
bugfix

Xlythe committed:
cleaned code

Xlythe committed:
tweak padding

Xlythe committed:
defaults for matrix keys

Xlythe committed:
added loading views

Xlythe committed:
fix removing views

Xlythe committed:
matrices displayed in display

Xlythe committed:
incrementally adding features

Xlythe committed:
cleaned up code

scrollable display is still broken

Xlythe committed:
fix unwanted edittexts

Xlythe committed:
code cleanup

Xlythe committed:
code cleanup

Xlythe committed:
make views easier to add

Xlythe committed:
remove wolfram code

Xlythe committed:
fix padding horror

Xlythe committed:
bugfixes

Xlythe committed:
framework for directing "next" button

Xlythe committed:
fix next button

Xlythe committed:
next button on CalcEditText

Xlythe committed:
solve matrices (basic logic)

Xlythe committed:
error check matrices

Xlythe committed:
always draw cursor

Xlythe committed:
use density for colorbutton hints

Xlythe committed:
bugfix

Xlythe committed:
fix longclick on display

Xlythe committed:
bugfixes

Xlythe committed:
translation updates from CyanogenMod

Xlythe committed:
bugfix

Xlythe committed:
framework for fixing Gravity.RIGHT

Xlythe committed:
tutorial added

Xlythe committed:
tutorial added (missing files)

Xlythe committed:
tutorial refined

Xlythe committed:
remove unneeded code

Xlythe committed:
refactored code

Xlythe committed:
change inverse formatting

Xlythe committed:
another tweak

Xlythe committed:
formatting

Xlythe committed:
fix horizontal scroll view

Xlythe committed:
keep cursor steady

Xlythe committed:
bugfixes

Xlythe committed:
bugfix

Xlythe committed:
fix autofilling blank matrices as 0

Xlythe committed:
bugfixs / tweaks with Next button

Xlythe committed:
code refactor

Xlythe committed:
make matrices smarter

bugfix: when copying from history, quotation mark was misplaced

Xlythe committed:
get rid of dictionary warning

graph buttons (in landscape) are on the far side

Xlythe committed:
only call onLayout when needed

Xlythe committed:
updated readme

Xlythe committed:
lower hint height

Xlythe committed:
cleaning up code

Xlythe committed:
dictionary warning back, but exponents fixed

Xlythe committed:
true fix for dictionary warning

Xlythe committed:
cling improvements

Xlythe committed:
ldpi was unncessary

Xlythe committed:
updated matrix 9patch

Xlythe committed:
set minHeight on widget display

Xlythe committed:
updated Slider

Xlythe committed:
fix display not expanding on tablets

Xlythe committed:
fix arrayoutofbounds error

Xlythe committed:
null pointers, go away

Xlythe committed:
catch illegal matrix operations

Xlythe committed:
add padding to display

Xlythe committed:
adjust text size of some buttons

Xlythe committed:
size readjustments

Xlythe committed:
remove unneeded code

Xlythe committed:
switch graph color to Android 4.0 blue

Xlythe committed:
dont graph matrices

Xlythe committed:
adjust sizes

Xlythe committed:
enable/disable views

Xlythe committed:
reverting part of the last commit

Xlythe committed:
center matrices, rearrange matrix page

sign button, easter egg

Xlythe committed:
tweaks

Xlythe committed:
switch to sp

Xlythe committed:
max display height

Xlythe committed:
smaller graphics for hdpi/mdpi

Xlythe committed:
use attrs for max display height

Xlythe committed:
fix crash

Xlythe committed:
xhdpi / mdpi assets resized

Xlythe committed:
better matrix validation

Xlythe committed:
removed 0-9, A-F, . from language strings

bugfix with pasting large equations into the calculator

Xlythe committed:
catch errors in the graph thread

Xlythe committed:
String tweaks thanks to Marco Brohet

Xlythe committed:
file deliminators in unix format

Xlythe committed:
bump up display height on tablets

Xlythe committed:
windows mode on samsung devices

matrices work in hexdecimal and binary now

Xlythe committed:
cleaning things up, tweaking res values

Xlythe committed:
more code cleanup

Xlythe committed:
height check didnt work, so reverted

Xlythe committed:
reset paint when button layout changes

Xlythe committed:
about page

Xlythe committed:
it translations from Francesco Foresta

Xlythe committed:
version number bump

Xlythe committed:
tweak wording on warning

Xlythe committed:
fix cutting

Xlythe committed:
dont comment out lines I dont mean to

Xlythe committed:
make android version a string

Xlythe committed:
fix for display

Xlythe committed:
preferences match trebuchet

Xlythe committed:
translations from trebuchet

Xlythe committed:
another fix for display

Xlythe committed:
not too fond of the about page

Xlythe committed:
fix tests

Xlythe committed:
pressed background for menu button

Xlythe committed:
another fix

Xlythe committed:
update version number

Xlythe committed:
menu background tweaks

Xlythe committed:
remove fade animation (buggy)

Xlythe committed:
background as theme

Xlythe committed:
asin, acos, atan

Xlythe committed:
translations from CyanogenMod

Xlythe committed:
Revert "asin, acos, atan"

This reverts commit 60a3ce15e16d7ad42168eea3677b65f7bcd78cfb.

Xlythe committed:
remove useless strings

Xlythe committed:
remove circle background from menu

Xlythe committed:
switch to about instead of application

Xlythe committed:
update translations

Xlythe committed:
update translations

Xlythe committed:
version bump

Xlythe committed:
fix decimals in matrices in some languages

note: should probably limit the decimal part of each value to 6 digits.
dont have a good way to do that though

Xlythe committed:
digit grouping

Xlythe committed:
widgets have unique ids

Xlythe committed:
widget saves to history

Xlythe committed:
split logic into modules

Xlythe committed:
disable buttons (and give disabled state)

Xlythe committed:
dynamically add comas

Xlythe committed:
allow functions in matrices

Xlythe committed:
clear and delete in wiget

Xlythe committed:
translation updates from cyanogenmod

Xlythe committed:
updated disabled state image

Xlythe committed:
disable buttons on tablets

Xlythe committed:
fix for base switching on tablets

Xlythe committed:
fix auto clear with number

Xlythe committed:
vibrate when del long pressed

Xlythe committed:
shorten vibration time

Xlythe committed:
remove vibration

Xlythe committed:
fix for null pointer when saving

Xlythe committed:
proper fix for auto clear with number

Xlythe committed:
vibrate on hold delete, ^! can be applied to results

Xlythe committed:
lock/unlock graph page

Xlythe committed:
fix adding ^!% operations

Xlythe committed:
fix comment

Xlythe committed:
use google's history adapter

Xlythe committed:
fix bugs with new adapter

Xlythe committed:
click to open history (and bugfixes)

Xlythe committed:
swipe to dismiss history

Xlythe committed:
shrink exponents once

Xlythe committed:
load fragment programmatically

Xlythe committed:
fix conflicts

Xlythe committed:
remove swipe to dismiss

Xlythe committed:
fix bugs with equation formatter

Xlythe committed:
fix deleting arcsin, cos, tan

Xlythe committed:
remove logging

Xlythe committed:
misc

Xlythe committed:
update slider

Xlythe committed:
more slider!

Xlythe committed:
fix negative numbers in matrices

Xlythe committed:
translation updates from cyanogenmod

Xlythe committed:
bump version

Xlythe committed:
start of themeing engine

Xlythe committed:
themeing engine almost complete

Xlythe committed:
bugfixes for themes

Xlythe committed:
themes for tablets

Xlythe committed:
add jruesga's cleanup

Xlythe committed:
catch crash on widget lockscreen paste

Xlythe committed:
added scaler logic for matrices (from Rudy Raab).

Added +, / to matrix page
Added Back to matrix page
Catch leading operations like (+, *)

Xlythe committed:
reformat xml files

Xlythe committed:
more xml cleanup

Xlythe committed:
rudy's fixes for matrix math

TransientResponse committed:
Fixed main.xml; un-broke matrix-matrix ops

TransientResponse committed:
Added matrix-scalar power. No order of operations though.

Xlythe committed:
mdpi, hdpi images

Xlythe committed:
support for comas as decimal points

Xlythe committed:
fix widget layout

Xlythe committed:
bugfixes for widget

Xlythe committed:
simplify code

TransientResponse committed:
Rewrote matrix math parsing. Now has full order of operations, the works. Only thing missing is functions (log, exp, etc)

Xlythe committed:
custom app icons

Xlythe committed:
bugfix for arc functions

Xlythe committed:
delete unused images

Xlythe committed:
themed menu pressed state

Xlythe committed:
refactoring

Xlythe committed:
update app icon

Xlythe committed:
tablet del button themed

Xlythe committed:
fix tablet images

TransientResponse committed:
Reworked parsing, much more efficient. Now also handles negative numbers (I hope) and functions.

TransientResponse committed:
Fixed matrix-scalar division and subtraction. Square root added.

TransientResponse committed:
Re-added matrix inverse and transpose. Unbalanced parens now throws exception. Fixed matrix-scalar division.

Xlythe committed:
fix for custom decimal points (ie: comas in french)

TransientResponse committed:
Fixed things, added math inside matrix declarations, and added factorials.

Xlythe committed:
localize matrix stuffs

TransientResponse committed:
Fixed order of operations AND improved performance. And added modulus.

TransientResponse committed:
Re-enabled try-catch to prevent outright crashing. A few minor changes.

Xlythe committed:
-, / in matrix page

TransientResponse committed:
Minor fixes, including to multiplication/division.

TransientResponse committed:
Made % percentage rather than mod. Added matrix determinant. Fixed scientific notation.

Xlythe committed:
revert slider changes

Xlythe committed:
fix my own stupidity in matrixs

Xlythe committed:
bugfixes

TransientResponse committed:
Minor fixes.

cm-gerrit commented:
Hi there;

This is an automated reply, to let you know CyanogenMod doesn't merge pull
requests submitted directly to our github.com repositories.

You can submit your change requests through our public review system,
located at http://review.cyanogenmod.org/

That review system is an installation of Google's Gerrit code review.
If you've never interacted with Gerrit before, we also suggest you take
a look at http://wiki.cyanogenmod.org/wiki/Howto:_Gerrit

Thanks!

On Wed, 18 Sep 2013 13:02:34 -0700 you wrote:

> I HAVE NO IDEA WHAT I&#39;M DOING
> You can merge this Pull Request by running:
> 
>   git pull https://github.com/TransientResponse/android_packages_apps_Calculator ics
> 
> Or you can view, comment on it, or merge it online at:
> 
>   https://github.com/CyanogenMod/android_packages_apps_Calculator/pull/7
> 
> -- Commit Summary --
> - removed tabs
> - translation tweaks
> - removed deprecated code
> - removed blank line on hex panel
> - updated slider
> - images update
> - more images
> - even more image updates
> - library update
> - formatting
> - code refactoring
> - Fix base conversion for numbers &gt;=10^7.
> - Merge pull request #4 from whydoubt/ics
> - negatives, decimals in matrices
> - tweaks to portrait phone, graphing logic
> - updated readme with descriptions of jar files
> - translation updates from CyanogenMod
> - disable keys in bin/dec
> - bugfixes
> - bugfix
> - bugfix
> - translation updates from CyanogenMod
> - bump version number
> - bugfix
> - degrees / radians option
> - degrees / radians enabled on tablets
> - Russian translation update thanks to Konstantin Vavilin
> - Start history at bottom
> - fix initial scroll
> - load history on startup
> - remove unneeded options when history is open/closed
> - launcher / lockscreen widget
> - widget bugfixes
> - exponents look exponently
> - Add a hint to display toasts
> - lockscreen widget fixed
> - follow widget standards for 4x4
> - lockscreen widget really fixed (I hope)
> - cleaning up code
> - custom zoom buttons
> - zoom buttons on all screen sizes
> - sec, csc, cot on long press of sin, cos, tan
> - display sec, csc, cot hints
> - bugfix
> - bugfix
> - bugfix
> - allow option of % and ()
> - reverted, % is more popular. dont need to complicate things
> - fix hint
> - meant inverse sin, inverse cos, inverse tan
> - remove dev logging
> - updated history look and feel
> - simplified code (made a text formatter class)
> - use tag for long press secondary feature
> - attempt to stop panning / zooming crashes
> - bugfix
> - save cycles - append ) when saving
> - bugfix
> - version bump
> - remove old matrix code
> - add new matrix page
> - separate views into own package
> - bugfix
> - fix preferences
> - move tablet xml from generic to port
> - remove deprecated code
> - hide display on matrix page
> - switch to matrix color buttons
> - wireframe of logic
> - new design should allow views in edittext
> - move copying
> - automatically display hints
> - bugfix for tablets
> - bugfix
> - cleaned code
> - tweak padding
> - defaults for matrix keys
> - added loading views
> - fix removing views
> - matrices displayed in display
> - incrementally adding features
> - cleaned up code
> - fix unwanted edittexts
> - code cleanup
> - code cleanup
> - make views easier to add
> - remove wolfram code
> - fix padding horror
> - bugfixes
> - framework for directing &quot;next&quot; button
> - fix next button
> - next button on CalcEditText
> - solve matrices (basic logic)
> - error check matrices
> - always draw cursor
> - use density for colorbutton hints
> - bugfix
> - fix longclick on display
> - bugfixes
> - translation updates from CyanogenMod
> - bugfix
> - framework for fixing Gravity.RIGHT
> - tutorial added
> - tutorial added (missing files)
> - tutorial refined
> - remove unneeded code
> - refactored code
> - change inverse formatting
> - another tweak
> - formatting
> - fix horizontal scroll view
> - keep cursor steady
> - bugfixes
> - bugfix
> - fix autofilling blank matrices as 0
> - bugfixs / tweaks with Next button
> - code refactor
> - make matrices smarter
> - get rid of dictionary warning
> - only call onLayout when needed
> - updated readme
> - lower hint height
> - cleaning up code
> - dictionary warning back, but exponents fixed
> - true fix for dictionary warning
> - cling improvements
> - ldpi was unncessary
> - updated matrix 9patch
> - set minHeight on widget display
> - updated Slider
> - fix display not expanding on tablets
> - fix arrayoutofbounds error
> - null pointers, go away
> - catch illegal matrix operations
> - add padding to display
> - adjust text size of some buttons
> - size readjustments
> - remove unneeded code
> - switch graph color to Android 4.0 blue
> - dont graph matrices
> - adjust sizes
> - enable/disable views
> - reverting part of the last commit
> - center matrices, rearrange matrix page
> - tweaks
> - switch to sp
> - max display height
> - smaller graphics for hdpi/mdpi
> - use attrs for max display height
> - fix crash
> - xhdpi / mdpi assets resized
> - better matrix validation
> - removed 0-9, A-F, . from language strings
> - catch errors in the graph thread
> - String tweaks thanks to Marco Brohet
> - file deliminators in unix format
> - bump up display height on tablets
> - windows mode on samsung devices
> - cleaning things up, tweaking res values
> - more code cleanup
> - height check didnt work, so reverted
> - reset paint when button layout changes
> - about page
> - it translations from Francesco Foresta
> - version number bump
> - tweak wording on warning
> - fix cutting
> - dont comment out lines I dont mean to
> - make android version a string
> - fix for display
> - preferences match trebuchet
> - translations from trebuchet
> - another fix for display
> - not too fond of the about page
> - fix tests
> - pressed background for menu button
> - another fix
> - update version number
> - menu background tweaks
> - remove fade animation (buggy)
> - background as theme
> - asin, acos, atan
> - translations from CyanogenMod
> - Revert &quot;asin, acos, atan&quot;
> - remove useless strings
> - remove circle background from menu
> - switch to about instead of application
> - update translations
> - update translations
> - version bump
> - fix decimals in matrices in some languages
> - digit grouping
> - widgets have unique ids
> - widget saves to history
> - split logic into modules
> - disable buttons (and give disabled state)
> - dynamically add comas
> - allow functions in matrices
> - clear and delete in wiget
> - translation updates from cyanogenmod
> - updated disabled state image
> - disable buttons on tablets
> - fix for base switching on tablets
> - fix auto clear with number
> - vibrate when del long pressed
> - shorten vibration time
> - remove vibration
> - fix for null pointer when saving
> - proper fix for auto clear with number
> - vibrate on hold delete, ^! can be applied to results
> - lock/unlock graph page
> - fix adding ^!% operations
> - fix comment
> - use google&#39;s history adapter
> - fix bugs with new adapter
> - click to open history (and bugfixes)
> - swipe to dismiss history
> - shrink exponents once
> - load fragment programmatically
> - fix conflicts
> - remove swipe to dismiss
> - fix bugs with equation formatter
> - fix deleting arcsin, cos, tan
> - remove logging
> - misc
> - update slider
> - more slider!
> - fix negative numbers in matrices
> - translation updates from cyanogenmod
> - bump version
> - start of themeing engine
> - themeing engine almost complete
> - bugfixes for themes
> - themes for tablets
> - add jruesga&#39;s cleanup
> - catch crash on widget lockscreen paste
> - added scaler logic for matrices (from Rudy Raab).
> - reformat xml files
> - Fixed main.xml; un-broke matrix-matrix ops
> - Added matrix-scalar power. No order of operations though.
> - Rewrote matrix math parsing. Now has full order of operations, the works. Only thing missing is functions (log, exp, etc)
> - Reworked parsing, much more efficient. Now also handles negative numbers (I hope) and functions.
> - Fixed matrix-scalar division and subtraction. Square root added.
> - Re-added matrix inverse and transpose. Unbalanced parens now throws exception. Fixed matrix-scalar division.
> - Fixed things, added math inside matrix declarations, and added factorials.
> - Fixed order of operations AND improved performance. And added modulus.
> - Re-enabled try-catch to prevent outright crashing. A few minor changes.
> - Minor fixes, including to multiplication/division.
> - Made % percentage rather than mod. Added matrix determinant. Fixed scientific notation.
> - Minor fixes.
> 
> -- File Changes --
> 
> ```
> A .classpath (9)
> A .gitignore (29)
> A .project (33)
> A .settings/org.eclipse.jdt.core.prefs (4)
> M Android.mk (7)
> M AndroidManifest.xml (30)
> A README.md (30)
> D arity-2.1.2.jar (0)
> A default.properties (11)
> A libs/achartengine.jar (0)
> A libs/android-support-v4.jar (0)
> A libs/arity-2.1.6.jar (0)
> A libs/ejml-0.21.jar (0)
> A libs/slider.jar (0)
> A project.properties (14)
> A res/color/text_color.xml (20)
> A res/color/text_color_light.xml (20)
> A res/drawable-hdpi/Thumbs.db (0)
> A res/drawable-hdpi/bg_cling.png (0)
> A res/drawable-hdpi/btn_cling_normal.9.png (0)
> A res/drawable-hdpi/btn_cling_pressed.9.png (0)
> A res/drawable-hdpi/cling.png (0)
> A res/drawable-hdpi/egg.png (0)
> A res/drawable-hdpi/egg_light.png (0)
> A res/drawable-hdpi/function.png (0)
> A res/drawable-hdpi/graph.png (0)
> A res/drawable-hdpi/graph_minus.png (0)
> A res/drawable-hdpi/graph_minus_light.png (0)
> A res/drawable-hdpi/graph_plus.png (0)
> A res/drawable-hdpi/graph_plus_light.png (0)
> A res/drawable-hdpi/graph_standard.png (0)
> A res/drawable-hdpi/graph_standard_light.png (0)
> A res/drawable-hdpi/hand.png (0)
> A res/drawable-hdpi/hex.png (0)
> A res/drawable-hdpi/history_divider.png (0)
> A res/drawable-hdpi/ic_menu_overflow_light.png (0)
> A res/drawable-hdpi/lock.png (0)
> A res/drawable-hdpi/matrix.png (0)
> A res/drawable-hdpi/matrix_background.9.png (0)
> A res/drawable-hdpi/settings.png (0)
> A res/drawable-hdpi/unlock.png (0)
> A res/drawable-mdpi/Thumbs.db (0)
> A res/drawable-mdpi/bg_cling.png (0)
> A res/drawable-mdpi/btn_cling_normal.9.png (0)
> A res/drawable-mdpi/btn_cling_pressed.9.png (0)
> A res/drawable-mdpi/btn_keyboard_key_light_disabled_holo.9.png (0)
> A res/drawable-mdpi/cling.png (0)
> A res/drawable-mdpi/egg.png (0)
> A res/drawable-mdpi/egg_light.png (0)
> A res/drawable-mdpi/function.png (0)
> A res/drawable-mdpi/graph.png (0)
> A res/drawable-mdpi/graph_minus.png (0)
> A res/drawable-mdpi/graph_minus_light.png (0)
> A res/drawable-mdpi/graph_plus.png (0)
> A res/drawable-mdpi/graph_plus_light.png (0)
> A res/drawable-mdpi/graph_standard.png (0)
> A res/drawable-mdpi/graph_standard_light.png (0)
> A res/drawable-mdpi/hand.png (0)
> A res/drawable-mdpi/hex.png (0)
> A res/drawable-mdpi/history_divider.png (0)
> A res/drawable-mdpi/ic_menu_overflow_light.png (0)
> A res/drawable-mdpi/lock.png (0)
> A res/drawable-mdpi/matrix.png (0)
> A res/drawable-mdpi/matrix_background.9.png (0)
> A res/drawable-mdpi/settings.png (0)
> A res/drawable-mdpi/unlock.png (0)
> A res/drawable-sw600dp-hdpi/Thumbs.db (0)
> A res/drawable-xhdpi/Thumbs.db (0)
> A res/drawable-xhdpi/advanced.png (0)
> A res/drawable-xhdpi/bg_cling.png (0)
> A res/drawable-xhdpi/btn_cling_normal.9.png (0)
> A res/drawable-xhdpi/btn_cling_pressed.9.png (0)
> A res/drawable-xhdpi/btn_keyboard_key_dark_normal_holo_light.9.png (0)
> A res/drawable-xhdpi/btn_keyboard_key_dark_pressed_holo_light.9.png (0)
> A res/drawable-xhdpi/btn_keyboard_key_equal_holo_light.9.png (0)
> A res/drawable-xhdpi/btn_keyboard_key_light_disabled_holo.9.png (0)
> A res/drawable-xhdpi/btn_keyboard_key_light_disabled_holo_light.9.png (0)
> A res/drawable-xhdpi/btn_keyboard_key_light_normal_holo_light.9.png (0)
> A res/drawable-xhdpi/btn_keyboard_key_light_pressed_holo_light.9.png (0)
> A res/drawable-xhdpi/btn_slider_holo_light.9.png (0)
> A res/drawable-xhdpi/cling.png (0)
> A res/drawable-xhdpi/egg.png (0)
> A res/drawable-xhdpi/egg_light.png (0)
> A res/drawable-xhdpi/function.png (0)
> A res/drawable-xhdpi/graph.png (0)
> A res/drawable-xhdpi/graph_minus.png (0)
> A res/drawable-xhdpi/graph_minus_light.png (0)
> A res/drawable-xhdpi/graph_plus.png (0)
> A res/drawable-xhdpi/graph_plus_light.png (0)
> A res/drawable-xhdpi/graph_standard.png (0)
> A res/drawable-xhdpi/graph_standard_light.png (0)
> A res/drawable-xhdpi/hand.png (0)
> A res/drawable-xhdpi/hex.png (0)
> A res/drawable-xhdpi/ic_menu_overflow_light.png (0)
> A res/drawable-xhdpi/lock.png (0)
> A res/drawable-xhdpi/matrix.png (0)
> A res/drawable-xhdpi/matrix_background.9.png (0)
> A res/drawable-xhdpi/settings.png (0)
> A res/drawable-xhdpi/simple.png (0)
> A res/drawable-xhdpi/unlock.png (0)
> M res/drawable/btn_digit.xml (12)
> A res/drawable/btn_digit_light.xml (21)
> A res/drawable/btn_equal_light.xml (21)
> M res/drawable/btn_function.xml (8)
> A res/drawable/btn_function_light.xml (22)
> A res/drawable/btn_slider_light.xml (21)
> A res/drawable/cling_button_bg.xml (20)
> A res/drawable/menu_bg.xml (21)
> M res/layout-land/advanced_pad.xml (110)
> A res/layout-land/function_pad.xml (75)
> A res/layout-land/graph_cling.xml (46)
> A res/layout-land/graph_pad.xml (35)
> A res/layout-land/hex_cling.xml (46)
> A res/layout-land/hex_pad.xml (80)
> D res/layout-land/history_item.xml (26)
> M res/layout-land/main.xml (161)
> A res/layout-land/matrix_cling.xml (46)
> A res/layout-land/matrix_pad.xml (148)
> A res/layout-land/simple_cling.xml (46)
> M res/layout-land/simple_pad.xml (112)
> M res/layout-port/advanced_pad.xml (122)
> A res/layout-port/function_pad.xml (93)
> A res/layout-port/graph_cling.xml (46)
> A res/layout-port/graph_pad.xml (36)
> A res/layout-port/hex_cling.xml (46)
> A res/layout-port/hex_pad.xml (81)
> D res/layout-port/history_item.xml (26)
> M res/layout-port/main.xml (179)
> A res/layout-port/matrix_cling.xml (46)
> A res/layout-port/matrix_pad.xml (161)
> A res/layout-port/simple_cling.xml (46)
> M res/layout-port/simple_pad.xml (106)
> A res/layout-sw600dp-land/advanced_pad.xml (100)
> A res/layout-sw600dp-land/function_pad.xml (93)
> A res/layout-sw600dp-land/graph_cling.xml (46)
> A res/layout-sw600dp-land/graph_pad.xml (35)
> A res/layout-sw600dp-land/hex_cling.xml (46)
> A res/layout-sw600dp-land/hex_pad.xml (81)
> M res/layout-sw600dp-land/main.xml (315)
> A res/layout-sw600dp-land/matrix_cling.xml (46)
> A res/layout-sw600dp-land/matrix_pad.xml (165)
> A res/layout-sw600dp-land/simple_cling.xml (46)
> A res/layout-sw600dp-land/simple_pad.xml (122)
> A res/layout-sw600dp-port/advanced_pad.xml (99)
> A res/layout-sw600dp-port/function_pad.xml (75)
> A res/layout-sw600dp-port/graph_cling.xml (46)
> A res/layout-sw600dp-port/graph_pad.xml (36)
> A res/layout-sw600dp-port/hex_cling.xml (46)
> A res/layout-sw600dp-port/hex_pad.xml (76)
> A res/layout-sw600dp-port/main.xml (124)
> A res/layout-sw600dp-port/matrix_cling.xml (46)
> A res/layout-sw600dp-port/matrix_pad.xml (167)
> A res/layout-sw600dp-port/simple_cling.xml (46)
> A res/layout-sw600dp-port/simple_pad.xml (122)
> D res/layout-sw600dp/main.xml (233)
> A res/layout-sw600dp/preferences.xml (66)
> A res/layout/history_entry.xml (19)
> A res/layout/history_panel.xml (4)
> A res/layout/preferences.xml (70)
> A res/layout/widget.xml (184)
> D res/menu-sw600dp/menu.xml (27)
> M res/menu/menu.xml (46)
> A res/values-af/cm_strings.xml (74)
> M res/values-am/strings.xml (90)
> A res/values-ar/cm_strings.xml (47)
> M res/values-ar/strings.xml (90)
> M res/values-be/strings.xml (90)
> M res/values-bg/strings.xml (90)
> A res/values-cs/cm_strings.xml (66)
> A res/values-da/cm_strings.xml (74)
> M res/values-da/strings.xml (4)
> A res/values-de/cm_strings.xml (77)
> M res/values-de/strings.xml (4)
> A res/values-el/cm_strings.xml (73)
> M res/values-el/strings.xml (90)
> A res/values-es/cm_strings.xml (77)
> M res/values-es/strings.xml (2)
> M res/values-fa/strings.xml (6)
> A res/values-fi/cm_strings.xml (72)
> A res/values-fr/cm_strings.xml (77)
> M res/values-fr/strings.xml (2)
> M res/values-hi/strings.xml (90)
> A res/values-hu/cm_strings.xml (77)
> A res/values-in/cm_strings.xml (47)
> M res/values-in/strings.xml (2)
> A res/values-it/cm_strings.xml (74)
> A res/values-iw/cm_strings.xml (49)
> M res/values-iw/strings.xml (90)
> A res/values-ja/cm_strings.xml (18)
> M res/values-ja/strings.xml (90)
> M res/values-ko/strings.xml (90)
> D res/values-land/arrays.xml (54)
> A res/values-nl/cm_strings.xml (77)
> M res/values-nl/strings.xml (2)
> A res/values-pl/cm_strings.xml (73)
> D res/values-port/arrays.xml (54)
> A res/values-port/styles.xml (19)
> A res/values-pt-rPT/cm_strings.xml (18)
> A res/values-pt/cm_strings.xml (77)
> M res/values-pt/strings.xml (2)
> M res/values-rm/strings.xml (2)
> A res/values-ro/cm_strings.xml (28)
> A res/values-ru/cm_strings.xml (59)
> M res/values-ru/strings.xml (91)
> M res/values-sr/strings.xml (90)
> A res/values-sw600dp-land/styles.xml (26)
> M res/values-sw600dp-port/styles.xml (1)
> D res/values-sw600dp/arrays.xml (51)
> M res/values-sw600dp/styles.xml (7)
> M res/values-th/strings.xml (90)
> M res/values-uk/strings.xml (90)
> A res/values-zh-rCN/cm_strings.xml (56)
> M res/values-zh-rCN/strings.xml (90)
> M res/values-zh-rTW/strings.xml (90)
> A res/values/attrs.xml (44)
> A res/values/cm_strings.xml (152)
> M res/values/colors.xml (13)
> M res/values/constants.xml (12)
> M res/values/strings.xml (13)
> M res/values/styles.xml (252)
> A res/xml/calculator_widget_info.xml (8)
> D src/com/android/calculator2/Calculator.java (341)
> D src/com/android/calculator2/CalculatorDisplay.java (165)
> D src/com/android/calculator2/CalculatorEditText.java (186)
> D src/com/android/calculator2/ColorButton.java (137)
> D src/com/android/calculator2/EventListener.java (136)
> D src/com/android/calculator2/Logic.java (318)
> D src/com/android/calculator2/PanelSwitcher.java (180)
> A src/com/android2/calculator3/BaseModule.java (340)
> A src/com/android2/calculator3/Calculator.java (711)
> R src/com/android2/calculator3/CalculatorEditable.java (62)
> A src/com/android2/calculator3/CalculatorSettings.java (61)
> A src/com/android2/calculator3/CalculatorWidget.java (233)
> A src/com/android2/calculator3/EquationFormatter.java (99)
> A src/com/android2/calculator3/EventListener.java (393)
> A src/com/android2/calculator3/Graph.java (153)
> A src/com/android2/calculator3/GraphModule.java (204)
> R src/com/android2/calculator3/History.java (39)
> R src/com/android2/calculator3/HistoryAdapter.java (63)
> R src/com/android2/calculator3/HistoryEntry.java (25)
> A src/com/android2/calculator3/LargePageAdapter.java (174)
> A src/com/android2/calculator3/Logic.java (409)
> A src/com/android2/calculator3/MatrixModule.java (656)
> A src/com/android2/calculator3/MutableString.java (45)
> A src/com/android2/calculator3/PageAdapter.java (226)
> R src/com/android2/calculator3/Persist.java (52)
> A src/com/android2/calculator3/Preferences.java (37)
> A src/com/android2/calculator3/SmallPageAdapter.java (144)
> A src/com/android2/calculator3/view/AdvancedDisplay.java (297)
> A src/com/android2/calculator3/view/CalculatorDisplay.java (241)
> A src/com/android2/calculator3/view/CalculatorEditText.java (236)
> R src/com/android2/calculator3/view/CalculatorViewPager.java (40)
> A src/com/android2/calculator3/view/Cling.java (187)
> A src/com/android2/calculator3/view/ColorButton.java (208)
> A src/com/android2/calculator3/view/HistoryLine.java (116)
> A src/com/android2/calculator3/view/MatrixEditText.java (137)
> A src/com/android2/calculator3/view/MatrixInverseView.java (67)
> A src/com/android2/calculator3/view/MatrixTransposeView.java (66)
> A src/com/android2/calculator3/view/MatrixView.java (289)
> A src/com/android2/calculator3/view/PreferencesFragment.java (48)
> A src/com/android2/calculator3/view/ScrollableDisplay.java (118)
> M tests/AndroidManifest.xml (6)
> R tests/src/com/android2/calculator3/CalculatorHitSomeButtons.java (19)
> R tests/src/com/android2/calculator3/CalculatorInstrumentationTestRunner.java (4)
> R tests/src/com/android2/calculator3/tests/CalculatorLaunchPerformance.java (4)
> ```
> 
> -- Patch Links --
> 
> https://github.com/CyanogenMod/android_packages_apps_Calculator/pull/7.patch
> https://github.com/CyanogenMod/android_packages_apps_Calculator/pull/7.diff

TransientResponse committed:
More minor fixes.

TransientResponse committed:
Trying to fix git.

Xlythe committed:
fix errors

Xlythe committed:
assume matrixToString is in English (or will be translated)

Xlythe committed:
relocalize

TransientResponse committed:
Fixed matrix-matrix operations. Regex was too greedy; made it lazy.

TransientResponse committed:
Fixed matrix-matrix operations. Regex was too greedy; made it lazy.

Xlythe committed:
bugfixes and formatting

Xlythe committed:
translation updates

TransientResponse committed:
Minor tweaks to code, and fix for order of operations.

Xlythe committed:
apply det to landscape pages

Xlythe committed:
deleted unneeded test file

Xlythe committed:
fix for failing tests

Xlythe committed:
more refined testing (find buttons for the current page, swipe to page)

Xlythe committed:
long click buttons

Xlythe committed:
clean up test code

TransientResponse committed:
Added unit test for matrix multiplication.

TransientResponse committed:
Merged from upstream.

TransientResponse committed:
Added matrix mult test case.

Xlythe committed:
bugfixes for tests

TransientResponse committed:
Added new unit test and fixed matrixToString()

Xlythe committed:
small fixes for tests

clear cling (if enabled)
clear text (if already inputted)
go to correct panel (in case its out of order)

Xlythe committed:
bugfix for del button in light theme

TransientResponse committed:
Added new unit test and vastly improved parentheses resolution

TransientResponse committed:
Merge branch 'ics' of github.com:Xlythe/android_packages_apps_Calculator into ics

Important change to parentheses resolution-- major fixes

Xlythe committed:
update history text color

TransientResponse committed:
Re-added parentheses auto-balance (maybe), fixed last unit test.

Xlythe committed:
slider improvements

Xlythe committed:
turn sin^-1 into asin

Xlythe committed:
bugfixes for foreign languages

Xlythe committed:
fix tests for phones

Xlythe committed:
update gitignore

Xlythe committed:
update light theme disabled buttons

TransientResponse committed:
Changed function handling to account for new names, and future sind etc. functions

TransientResponse committed:
Merge branch 'ics' of github.com:Xlythe/android_packages_apps_Calculator into ics

Just get out of my way, github

cm-gerrit commented:
Hi there;

This is an automated reply, to let you know CyanogenMod doesn't merge pull
requests submitted directly to our github.com repositories.

You can submit your change requests through our public review system,
located at http://review.cyanogenmod.org/

That review system is an installation of Google's Gerrit code review.
If you've never interacted with Gerrit before, we also suggest you take
a look at http://wiki.cyanogenmod.org/wiki/Howto:_Gerrit

Thanks!
